### PR TITLE
PHPCS fixes: `Squiz`

### PR DESCRIPTION
**Pull request by crstauf**
Eliminates smells from the `Squiz` standard.

See https://github.com/woocommerce/action-scheduler/issues/971.

### CODE REVIEW

**Comment by crstauf** [comment]
@barryhughes More Composer failures.

File: classes/abstracts/ActionScheduler.php [hunk 1]
```diff
@@ -149,7 +173,8 @@ public static function init( $plugin_file ) {
 
 		// Ensure initialization on plugin activation.
 		if ( ! did_action( 'init' ) ) {
-			add_action( 'init', array( $admin_view, 'init' ), 0, 0 ); // run before $store::init()
+			// phpcs:ignore Squiz.PHP.CommentedOutCode
```
Comment (barryhughes) [inline classes/abstracts/ActionScheduler.php:176]: Is this line required (which commented out code does it refer to)?

```suggestion
```

Comment (crstauf) [reply classes/abstracts/ActionScheduler.php:176]: `// run before $store::init().`

File: classes/data-stores/ActionScheduler_wpPostStore_PostStatusRegistrar.php [hunk 2]
```diff
@@ -5,6 +5,10 @@
  * @codeCoverageIgnore
  */
 class ActionScheduler_wpPostStore_PostStatusRegistrar {
+
+	/**
+	 * Registrar.
```
Comment (barryhughes) [inline classes/data-stores/ActionScheduler_wpPostStore_PostStatusRegistrar.php:10]: Bit of a nit, doesn't matter too much, but why *registrar* vs *register* (the latter matches the method name)?

File: classes/schema/ActionScheduler_StoreSchema.php [hunk 3]
```diff
@@ -33,10 +36,16 @@ public function init() {
...
+	/**
+	 * Get table definition.
+	 *
+	 * @param string $table Table name.
+	 */
 	protected function get_table_definition( $table ) {
 		global $wpdb;
 		$table_name       = $wpdb->$table;
 		$charset_collate  = $wpdb->get_charset_collate();
+		// phpcs:ignore Squiz.PHP.CommentedOutCode
```
Comment (barryhughes) [inline classes/schema/ActionScheduler_StoreSchema.php:48]: Similar to an earlier question: is this required/what does it refer to?

```suggestion
```

Comment (crstauf) [reply classes/schema/ActionScheduler_StoreSchema.php:48]: `// @see wp_get_db_schema()`

File: classes/data-stores/ActionScheduler_wpPostStore_PostStatusRegistrar.php (hunk 2 above)
Comment (crstauf) [reply classes/data-stores/ActionScheduler_wpPostStore_PostStatusRegistrar.php:10]: Matches the class name.

File: classes/abstracts/ActionScheduler.php (hunk 1 above)
Comment (barryhughes) [reply classes/abstracts/ActionScheduler.php:176]: Interesting. It isn't triggered for me (ie, I don't see an error for that rule, if the `phpcs:ignore` is removed).

Comment (barryhughes) [reply classes/abstracts/ActionScheduler.php:176]: (Though I would see it if I commented out the full line...) are you invoking phpcs via Composer (are you using the project's PHPCS)?

File: classes/schema/ActionScheduler_StoreSchema.php (hunk 3 above)
Comment (barryhughes) [reply classes/schema/ActionScheduler_StoreSchema.php:48]: I guess `// @see wp_get_db_schema()` on the next line, yet this rule violation is not reported when I run PHPCS (assuming that ignore statement isn't present).

**Review by barryhughes — APPROVED** [review]
Thanks! I have a couple of questions, but they are non-blocking (though I'm curious about the differences we are possibly seeing, in relation to commented out code).

We seem to be having some issues with CI right now, so bear with me while I look at that.

Otherwise, do you have a preference in terms of when this is merged (is it fine from your perspective if we merge this next, relative to your other PHPCS-related PR)?

**Comment by crstauf** [comment]
> Otherwise, do you have a preference in terms of when this is merged (is it fine from your perspective if we merge this next, relative to your other PHPCS-related PR)?

@barryhughes I think merging it before the PHPCS fixes makes sense.

**Comment by crstauf** [comment]
> Otherwise, do you have a preference in terms of when this is merged (is it fine from your perspective if we merge this next, relative to your other PHPCS-related PR)?

@barryhughes Yes, fine to do Squiz before WP/WC.

**Comment by barryhughes** [comment]
- Merged via https://github.com/woocommerce/action-scheduler/pull/1087
- Same set of commits.

**Pull request closed without merging**